stacks_queues/stack.py

```diff
@@ -11,28 +12,40 @@ def __init__(self):
     def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
+            stacks -> last in first out 
         """
-        pass
-
+        self.store.add_last(element)
+        return element
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
             returns None
         """
-        pass
+        if not self.store:
+            raise StackEmptyException("List is empty")
+
+        return self.store.remove_last()  
```

Comment:
Again, per the function specification, this method should return `None`
```suggestion
        self.store.remove_last()  
```